ide/extexecution.process/external/processtreekiller-2.0.1-license.txt

```diff
@@ -1,6 +1,6 @@
 Name: NetBeans processtreekiller
 Version: 2.0.1
-License: MIT
+License: MIT-processtreekiller
```

Comment:
MIT is a generic template of the license, without copyright notices. This file is correct, but we should have the correct file in the `nbbuild/licenses` database as well.

See the source license here:
https://github.com/matthiasblaesing/netbeans-processtreekiller/blob/master/LICENSE